doc/backlinks.rst

```diff
@@ -1,14 +0,0 @@
-
-Some projects using Pylint
```

Comment:
Removed because it's super outdated and we're not some kind of startup trying to secure funding so it's noise.